.github/actions/upgrade-testing/scripts/raise_gov_proposal.py

```diff
@@ -0,0 +1,25 @@
+import os
+import requests
+os.environ['NODE'] = "http://127.0.0.1:26657"
+CURRENT_HEIGHT = requests.get(f"{os.environ['NODE']}/status").json()["result"]["sync_info"]["latest_block_height"]
+UPGRADE_HEIGHT = int(CURRENT_HEIGHT) + (int(os.environ['PROPOSAL_TIME_SECONDS']) / int(os.environ['BLOCK_TIME_SECONDS'])) + 20
+github_file = open(os.environ["GITHUB_ENV"], "a+")
+github_file.write(f"UPGRADE_HEIGHT={UPGRADE_HEIGHT}")
+github_file.close()
+
+GOV_PROPOSAL = f"""zetacored tx gov submit-legacy-proposal software-upgrade "{os.environ['VERSION']}" \
```

Comment:
@gzukel  can we try moving away from the legacy proposal, and submit using the regular proposal instead?